src/main/java/br/com/deveficiente/calendario/controller/form/NotificacaoForm.java

```diff
@@ -0,0 +1,38 @@
+package br.com.deveficiente.calendario.controller.form;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+public class NotificacaoForm {
+
+    @NotEmpty(message = "Informe o tipo de notificação")
+    private String tipoNotificacao;
```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*

Aqui você pode receber a enum direto. Basta que o valor do parâmetro seja exatamente igual a enum e, no seu caso, já é :).